.eslintrc.cjs

```diff
@@ -0,0 +1,194 @@
+		"getter-return": 1, // Enforce `return` statements in getters
+		"no-async-promise-executor": 1, // Disallow using an async function as a Promise executor
+		"no-case-declarations": 1, // Disallow lexical declarations in case clauses
+		"no-class-assign": 1, // Disallow reassigning class members
+		"no-compare-neg-zero": 1, // Disallow comparing against -0
+		"no-const-assign": 1, // Disallow reassigning `const` variables
+		"no-constant-condition": 1, // Disallow constant expressions in conditions
+		"no-control-regex": 1, // Disallow control characters in regular expressions
+		"no-debugger": 1, // Disallow the use of `debugger`
+		"no-delete-var": 1, // Disallow deleting variables
+		"no-dupe-args": 1, // Disallow duplicate arguments in `function` definitions
+		"no-dupe-class-members": 1, // Disallow duplicate class members
+		"no-dupe-else-if": 1, // Disallow duplicate conditions in if-else-if chains
+		"no-dupe-keys": 1, // Disallow duplicate keys in object literals
+		"no-duplicate-case": 1, // Disallow duplicate case labels
+		"no-empty-character-class": 1, // Disallow empty character classes in regular expressions
+		"no-empty-pattern": 1, // Disallow empty destructuring patterns
+		"no-ex-assign": 1, // Disallow reassigning exceptions in `catch` clauses
+		"no-extra-boolean-cast": 1, // Disallow unnecessary boolean casts
+		"no-fallthrough": 1, // Disallow fallthrough of `case` statements
+		"no-func-assign": 1, // Disallow reassigning `function` declarations
+		"no-global-assign": 1, // Disallow assignments to native objects or read-only global variables
+		"no-import-assign": 1, // Disallow assigning to imported bindings
+		"no-inner-declarations": 1, // Disallow variable or `function` declarations in nested blocks
+		"no-invalid-regexp": 1, // Disallow invalid regular expression strings in `RegExp` constructors
+		"no-irregular-whitespace": 1, // Disallow irregular whitespace
+		"no-misleading-character-class": 1, // Disallow characters which are made with multiple code points in character class syntax
+		"no-new-symbol": 1, // Disallow `new` operators with the `Symbol` object
+		"no-nonoctal-decimal-escape": 1, // Disallow `\8` and `\9` escape sequences in string literals
+		"no-obj-calls": 1, // Disallow calling global object properties as functions
+		"no-octal": 1, // Disallow octal literals
+		"no-prototype-builtins": 1, // Disallow calling some `Object.prototype` methods directly on objects
+		"no-regex-spaces": 1, // Disallow multiple spaces in regular expressions
+		"no-self-assign": 1, // Disallow assignments where both sides are exactly the same
+		"no-shadow-restricted-names": 1, // Disallow identifiers from shadowing restricted names
+		"no-this-before-super": 1, // Disallow `this`/`super` before calling `super()` in constructors
+		"no-undef": 1, // Disallow the use of undeclared variables unless mentioned in `/*global */` comments
+		"no-unexpected-multiline": 1, // Disallow confusing multiline expressions
```

Comment:
I’m a little skeptical about this one; it could be good if it works well, but I'm a bit worried about false positives. Can we also mark it as at-risk?